databricks/koalas/indexing.py

```diff
@@ -31,28 +32,27 @@ def _make_col(c):
     if isinstance(c, Series):
         return c._scol
     elif isinstance(c, str):
-        from pyspark.sql.functions import col
-        return col(c)
+        return F.col(c)
     else:
         raise SparkPandasNotImplementedError(
             description="Can only convert a string to a column type.")
 
 
-def _unfold(key, col):
+def _unfold(key, ks):
     """ Return row selection and column selection pair.
 
-    If col parameter is not None, the key should be row selection and the column selection will be
-    the col parameter itself. Otherwise check the key contains column selection, and the selection
-    is acceptable.
+    If ks parameter is not None, the key should be row selection and the column selection will be
+    the Spark Column in the ks parameter. Otherwise check the key contains column selection, and
+    the selection is acceptable.
     """
```

Comment:
if you can do couple doctests here, i think it will make the function easier to understand (basically by giving examples).